SETUP.md

```diff
@@ -251,6 +252,9 @@ Note: Virtual Machine Users should check the [Alternative note](#alternative-use
     1. If your PATH is missing `~/.rbenv/shims`, the next two commands might not work. Edit your .bashrc to include the following line:
        `export PATH="$HOME/.rbenv/bin:~/.rbenv/shims:$PATH"`, then run `source .bashrc` for the change to take effect (as seen in [this github issue](https://github.com/rbenv/rbenv/issues/877)).
     1. `rbenv rehash`
+1. Install pdm, which will be used later by `rake install` to install python
+    1. `sudo pip3 install --prefix=/usr/local --upgrade pdm`
+        - alternatively, if you prefer pipx and have it configured path-wise: `pipx install pdm`
```

Comment:
This hasn't been working for me, not sure if it's the instructions or my environment.
![image](https://github.com/user-attachments/assets/06f0bd64-136f-4fba-8e2a-f5196c932b59)

Reply:
1. Which ubuntu? Maybe I can replicate in docker
2. Does `/usr/local/bin/pdm` exist? 
  - If so, is `/usr/local/bin` in your path?
    - If so, maybe you just need a `zsh rehash` to rescan the PATH?
3. What about `/usr/local/lib/python3.8/dist-packages/pdm`
  - it might be other than python3.8 in the path, esp if not on ubuntu 20.04: `ls /usr/local/lib/`
 4. If all else fails: `find /usr/local | grep pdm`, on 20.04 clean docker mine produces:
 ```
 circleci@75027a115ad9:/usr/local$ find /usr/local | grep pdm
 /usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/WHEEL
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/RECORD
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/INSTALLER
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/licenses
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/licenses/LICENSE
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/METADATA
/usr/local/lib/python3.8/dist-packages/pdm-2.17.3.dist-info/entry_points.txt
/usr/local/lib/python3.8/dist-packages/pdm
/usr/local/lib/python3.8/dist-packages/pdm/project
SNIP
/usr/local/lib/python3.8/dist-packages/pdm/resolver/core.py
/usr/local/lib/python3.8/dist-packages/pdm/resolver/__init__.py
/usr/local/lib/python3.8/dist-packages/pdm/resolver/providers.py
/usr/local/bin/pdm
```

Reply:
I ran into the same thing, and it was a path issue. Thanks Seth!